graphql/graphql-vs-openapi.md

```diff
@@ -0,0 +1,752 @@
+
+-  **IBM**: Implements OpenAPI for its cloud services, ensuring robust and reliable API interactions.
+
+-  **Google**: Adopts OpenAPI for its various APIs, emphasizing clear documentation and ease of use.
+
+  
+
+### B. Real-World Case Studies
+
+Exploring real-world case studies illustrates the practical benefits and challenges of using GraphQL and OpenAPI.
+  
+
+#### Success Stories and Lessons Learned
+
+1.  **GraphQL**:
+
+-   **Globoplay:** The Brazilian streaming service Globoplay leveraged GraphQL to unify its data across various platforms, enhancing customer experience and operational efficiency. By adopting Apollo's supergraph, they were able to create a single source of truth for their data, which reduced the maintenance burden and allowed faster feature releases​ .​​
+    
+-   **Volvo Cars:** Volvo implemented GraphQL to manage their online car shopping experience. The use of Apollo's supergraph enabled Volvo to create a unified data model, which streamlined their operations and improved the speed of new feature deployments​ .
+    
+-   **Priceline:** Priceline adopted GraphQL to modernize their API infrastructure. This transition allowed them to handle complex travel booking queries more efficiently, reducing response times and enhancing user experience. The move to GraphQL also facilitated better compliance with PCI standards​ .
+    
+-   **Hasura and Pipe:** Pipe, a fintech company, used Hasura's GraphQL engine to move from prototype to production in just 11 days. This rapid development cycle was crucial for their business, enabling them to quickly validate and launch new financial products​ .
+  
+
+2.  **OpenAPI**:
+
+-   **BBVA:** The Spanish banking giant BBVA utilized OpenAPI to build an open-source security product. By standardizing their API documentation and using OpenAPI's robust validation features, they enhanced security and compliance across their digital services.​ 
+    
+-   **Nutrien:** Nutrien, a global agricultural solutions provider, accelerated the development of their digital hub using OpenAPI. The clear structure and validation provided by OpenAPI ensured reliable data exchange and integration across their systems, streamlining operations and improving service delivery​.
+    
+-   **Philips Healthcare:** Philips leveraged OpenAPI to speed up development processes by up to 4 times. The standardized approach of OpenAPI facilitated better collaboration and integration across their various healthcare platforms, ensuring consistency and reliability in data exchange​ ​.
+    
+-   **Cox Automotive:** Cox Automotive transitioned from API silos to a unified API strategy with OpenAPI. This move allowed them to unlock APIs as economic assets, improving operational efficiency and providing a cohesive data strategy across their platforms.
+
+Sources : [([Hasura](https://hasura.io/customers)​),([Apollo](https://www.apollographql.com/customers))]
```

Comment:
Just keep links to the other blogs with case study

Reply:
made into case studies and kept links to the original blogs